Demo/Sources/Views/PlaybackSlider.swift

```diff
@@ -0,0 +1,118 @@
+//
+//  Copyright (c) SRG SSR. All rights reserved.
+//
+//  License information is available from the LICENSE file.
+//
+
+import Player
+import SwiftUI
+
+#if os(iOS)
+struct PlaybackSlider: View {
+    let progressTracker: ProgressTracker
+    let minimumValueText: String?
+    let maximumValueText: String?
+
+    @Binding private var value: Float
+    @State private var valueWidth: CGFloat = 0
+    @State private var previousValueWidth: CGFloat = 0
```

Comment:
Could we maybe get rid of this state? Not sure why this would be necessary if we can access the drag gesture start location.

Reply:
If we want the same behavior as the Apple Slider, we should keep the previous value. 
This will prevent jumping directly to the starting gesture location.